lib/assert.mjs

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-restricted-properties */
+import assert from 'assert';
+
+export default assert;
+
+export const fail = assert.fail;
+export const AssertionError = assert.AssertionError;
+export const ok = assert.ok;
+export const equal = assert.equal;
+export const notEqual = assert.notEqual;
+export const deepEqual = assert.deepEqual;
+export const notDeepEqual = assert.notDeepEqual;
+export const deepStrictEqual = assert.deepStrictEqual;
+export const notDeepStrictEqual = assert.notDeepStrictEqual;
+export const strictEqual = assert.strictEqual;
+export const notStrictEqual = assert.notStrictEqual;
+export const throws = assert.throws;
+export const doesNotThrow = assert.doesNotThrow;
+export const ifError = assert.ifError;
```

Comment:
This appears to be missing the new `strict` export.

Reply:
it might not have been in my branch when i generated these, but since i'm changing this to generate the exports at runtime it shouldn't be an issue. (stay tuned!! 😄)